srcpkgs/go-ethereum/template

```diff
@@ -12,13 +12,12 @@ ${go_import_path}/cmd/puppeth
 ${go_import_path}/cmd/rlpdump
 ${go_import_path}/cmd/wnode
 ${go_import_path}/cmd/swarm"
-depends="geth"
```

Comment:
@maxice8 do we need this?
The go-package already includes `${go_import_path}/cmd/geth`

Reply:
Yes we need that go-etherum depend on geth, we have split geth into a subpackage so the geth binary isn't in the main package.

Reply:
that makes sense.

Reply:
got it. Thanks for the explanation. I've fixed the PR.